src/mappings/content/utils.ts

```diff
@@ -712,49 +712,48 @@ export async function addNewBidNotification(
 ) {
   if (previousTopBid?.bidderId) {
     const outbiddedMemberId = previousTopBid.bidderId
-    const outbiddedMemberHandle = await memberHandleById(overlay, outbiddedMemberId)
     const outbiddedMemberAccount = await getAccountForMember(overlay.getEm(), outbiddedMemberId)
+
+    const notificationData = {
+      newBidderHandle: newTopBidderHandle,
+      videoId,
+      videoTitle,
+    }
     await addNotification(
       overlay.getEm(),
       outbiddedMemberAccount,
-      new HigherBidPlaced({
-        recipient: new MemberRecipient({ memberHandle: outbiddedMemberHandle }),
-        newBidderHandle: newTopBidderHandle,
-        videoId,
-        videoTitle,
-      }),
+      new MemberRecipient({ membership: outbiddedMemberId }),
+      new HigherBidPlaced(notificationData),
```

Comment:
I'll check this one too (also doesn't show ref https://github.com/Joystream/atlas/issues/4891#issuecomment-1725299795)

Reply:
This missing was probably due to the missing preference, so it returned the defaults:
 - `inApp:false , email:false`